### ISSUE TITLE
Displaying the basic premium Telegram icon in the Profile Menu

### ISSUE THREAD

**Issue by Dimoka113** [telegramdesktop/tdesktop#27259]
### Steps to reproduce

https://github.com/telegramdesktop/tdesktop/assets/133571582/6476802f-909d-4230-be63-a17bd5fb5fd2

### Expected behaviour

The Emoji status should change according to the selected.

### Actual behaviour

Nothing. The emoji changes, but I can't see it.

### Operating system

Windows 10

### Version of Telegram Desktop

4.13 x64

### Installation source

Static binary from official website

### Crash ID

No

### Logs

```shell
Zero information.
https://github.com/telegramdesktop/tdesktop/files/13759817/log.txt
```

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/commit/4aa8a4111981ce542cf35f38e3db52b41339dbc4

**Comment by john-preston** [comment]
@Dimoka113 Fixed in 4.13.1, will be released within several minutes for Windows and a bit later for macOS and Linux.

**Comment by Dimoka113** [comment]
> @Dimoka113Исправлено в версии 4.13.1, будет выпущено через несколько минут для Windows и чуть позже для macOS и Linux.

Thanks.
